backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -14,6 +14,7 @@
 import com.google.common.base.Strings;
 import com.google.common.collect.*;
 
+import org.apache.logging.log4j.LogManager;
```

Comment:
Would it make sense to have the imports sorted and close by each other? Please consider this in all cases.

Reply:
Logging-related imports have been ordered in all files that have been touched (unless I missed some ;-)